cmd/zpool/zpool_main.c

```diff
@@ -237,8 +255,9 @@ get_usage(zpool_help_t idx) {
 		    "[-R root] [-F [-n]]\n"
 		    "\t    <pool | id> [newpool]\n"));
 	case HELP_IOSTAT:
-		return (gettext("\tiostat [-gLPvy] [-T d|u] [pool] ... "
-		    "[interval [count]]\n"));
+		return (gettext("\tiostat [-ghHLpPvy] [[-lq]|-G] [-s name] "
+		    "[-T d|u] [-y]"
```

Comment:
The `[-y]` here is redundant and you're missing a `\n` before the `[pool...` chunk.